src/Sylius/Bundle/ApiBundle/SerializerContextBuilder/ParametersContextBuilder.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\SerializerContextBuilder;
+
+use ApiPlatform\Core\Metadata\Resource\Factory\ResourceMetadataFactoryInterface;
+use ApiPlatform\Core\Serializer\SerializerContextBuilderInterface;
+use Sylius\Bundle\ApiBundle\Metadata\Resource\AttributeKeys;
+use Sylius\Bundle\ResourceBundle\Controller\ParametersParserInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+final class ParametersContextBuilder implements SerializerContextBuilderInterface
```

Comment:
It is maybe a problem with my comprehension, but I would say that this class is not needed as well